docs/dsp-tools-create.md

```diff
@@ -426,7 +396,7 @@ example, the list "colors" could be imported as follows:
                 "en": "A list with categories"
             },
             "nodes": [
-                ...
```

Comment:
This and other similar changes are just that PyCharm doesn't show that many problems. In the future, I'd like to look more frequently on the problems that are detected by PyCharm. But for this, I need to kick out the false positives.